PD-1925 / 25.10 / PD-1925 Update Cloud Sync Task Screens Article (#3729)
truenas/documentation: content/SCALE/SCALEUIReference/DataProtection/CloudSyncTasksScreensSCALE.md

* The <span class="material-icons">play_circle_filled</span> **Run Now** icon starts the cloud sync, running it outside of the scheduled time. 

* The <span class="material-icons">loop</span> **Dry Run** icon performs the same function as the **Dry Run** button on the add and edit configuration screens.
  It tests and validates the configured settings.
  During a dry run, you can close the window and monitor the task using the **Jobs** option on the top toolbar.

* The <span class="material-icons">restore</span> **Restore** icon creates a new cloud sync task from an existing task.
  The new task has the same settings but reverses the data transfer.

* The <span class="material-icons">delete</span> **Delete** icon opens a confirmation dialog before the system deletes the task.

> **State** shows the status of the next cloud sync task.Statuse are:

Reviewer: ```suggestion
**State** shows the status of the next cloud sync task:
```

* **Delete** opens a dialog where you confirm the action before the system deletes the task.

## Cloudsync Task Wizard

The **Cloud Sync Task** wizard screens simplify the task creation process.
The wizard has two screens, **Provider** and **What and When**.

### Provider Wizard Screen

The **Provider** wizard screen allows you to select the cloud sync provider with the **Credentials** dropdown.

> Select the provider from the dropdown list to show the additional credential settings the seleCted provider requires to establish a connection.

Reviewer: ```suggestion
Select the provider from the dropdown list to show the additional credential settings the selected provider requires to establish a connection.
```

{{< truetable >}}
| Settings | Description |
|----------|-------------|

> | **Description** | A description of the cloud sync task entered into the field. |

Reviewer: Suggestion to rework the definition a little bit more informatively.
```suggestion
| **Description** | A human-readable summary about the task. This is for administrator reference only. |
```

> | **Direction** | Sets a direction option to what is selected from the dropdown list. **PUSH** sends data to cloud storage. **PULL** receives data from cloud storage and is the default setting. |

Reviewer: ```suggestion
| **Direction** | Defines whether TrueNAS sends or receives data. **PUSH** sends data to cloud storage. **PULL** receives data from cloud storage and is the default setting. |
```

> | **Transfer Mode** | Sets the transfer mode type to the option selected on the dropdown list. There are three options:<br><li>**COPY** - Select to duplicate each source file into the destination. If files with the same names are present on the destination, they are overwritten.<br><li>**MOVE** - Select to transfer files from the source to the destination and delete source files. Copies files from the source to the destination and then deletes them from the source. Files with the same names on the destination are overwritten.<br><li>**SYNC** - Select to change files on the destination to match those on the source. If a file does not exist on the source, it is also deleted from the destination.</li> |

Reviewer: ```suggestion
| **Transfer Mode** | Defines how files are handled during the task. There are three options:<br><li>**COPY** - Select to duplicate each source file into the destination. If files with the same names are present on the destination, they are overwritten.<br><li>**MOVE** - Select to transfer files from the source to the destination and delete source files. Copies files from the source to the destination and then deletes them from the source. Files with the same names on the destination are overwritten.<br><li>**SYNC** - Select to change files on the destination to match those on the source. If a file does not exist on the source, it is also deleted from the destination.</li> |
```

> | **Directory/Files** | Enter or browse to select the dataset or folder. Click the <span class="material-icons">arrow_right</span> arrow to the left of **/mnt** folder to expand and show datasets and directories within that folder. After selecting the dataset or directory location (source) to send to the cloud for push syncs, or as the destination to write to for pull syncs. Be cautious with pull destinations to avoid overwriting existing files. Click the <span class="material-icons">arrow_right</span> arrow to the left of **/mnt** folder again to collapse the directory tree. Note: Clear the checkmark to the left of **/mnt**. Not clearing this checkmark can result in an **All selected directories must be at the same level** error message. |

Reviewer: ```suggestion
| **Directory/Files** | Enter or browse to select the source or receiving dataset or folder. Click the <span class="material-icons">arrow_right</span> arrow to the left of **/mnt** folder to expand and show datasets and directories within that folder. After selecting the dataset or directory location (source) to send to the cloud for push syncs, or as the destination to write to for pull syncs. Be cautious with pull destinations to avoid overwriting existing files. Click the <span class="material-icons">arrow_right</span> arrow to the left of **/mnt** folder again to collapse the directory tree. Note: Clear the checkmark to the left of **/mnt**. Not clearing this checkmark can result in an **All selected directories must be at the same level** error message. |
```
